### ISSUE TITLE
Display settings mislabel monitors

### ISSUE THREAD

**Issue by richardcoffeyflock** [regolith-linux/regolith-desktop#622]
- [YES ] I've [run a search on the existing bug reports](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [YES ] I've read [the Regolith documentation](https://regolith-linux.org/docs/) and didn't find any relevant topic
- [YES ] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)

**Describe the bug**
In the "Displays" section of the settings, Regolith incorrectly labels the screens when using a multi-monitor setup. Please view attached picture, where screens 1 and 2 are incorrectly labeled as each other. The actual configuration of the the screens in the organization drag-n-drop GUI is correct, it's the on screen labeling of the monitors that is incorrect. Please note that screen #3 in the screenshot is rotated at a 90 degree angle to "Portrait Right"

**To Reproduce**
With 3 monitors, open the display settings page and hover over the organization drag-n-drop GUI. Each screen should have a numbered label pop up in the top left corner. These labels are incorrect often, and do not have a specific rhyme or reason. 

**Expected behavior**
The labels for the screens should be identical to the ones displayed in the GUI.

**Screenshots**
Attached.

**Configuration file(s)**
None.

**Installation Details**
 - Regolith Version: 1.6
 - Host OS Ubuntu 20.04
![mislabled_screen-min](https://user-images.githubusercontent.com/76059861/149541681-6b92485d-30ac-4118-ac24-a362d29d7f94.png)

**Comment by kgilmer** [comment]
@richardcoffeyflock yes I'm aware of this bug as well.  I believe it's an upstream issue in `gnome-flashback`: https://gitlab.gnome.org/GNOME/gnome-flashback, although w/ upcoming Regolith 2 we'll be owning this UI w/ a fork called `regolith-control-center`.  I'll leave this bug open to track a future fix in the 2.x release.

**Comment by rstub** [comment]
I see similar miss-labeling of screens using

```
ralf@barra:~$ dpkg -l regolith-control-center regolith-desktop 
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                    Version           Architecture Description
+++-=======================-=================-============-=============================================
ii  regolith-control-center 1.36.17-1regolith amd64        utilities to configure the Regolith desktop
ii  regolith-desktop        4.1.2-1-1regolith amd64        Meta package for Regolith desktop environment
ralf@barra:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
```

Example:
![image](https://user-images.githubusercontent.com/27821131/176924478-e624099e-f57d-4fb9-a290-dabdc046ae73.png)

**Comment by dalyndalton** [comment]
Bumping this bug to say that I'm still affected on flashback on the testing branch of noble.

**Comment by klew** [comment]
I have this issue on Regolith 3.2 with "regolith-control-center 46.beta".
